solutions/src/main/java/ru/sberbank/school/task06/Pet.java

```diff
@@ -0,0 +1,67 @@
+package ru.sberbank.school.task06;
+
+import java.util.Objects;
+
+public class Pet{
+    private String name;
+    private boolean sex;
+    private Integer age;
+    private String breed;
+
+    public Pet(String name, boolean sex, int age, String breed) {
+        this.name = name;
+        this.sex = sex;
+        this.age = age;
+        this.breed = breed;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public boolean isSex() {
+        return sex;
+    }
+
+    public void setSex(boolean sex) {
+        this.sex = sex;
+    }
+
+    public int getAge() {
+        return age;
+    }
+
+    public void setAge(int age) {
+        this.age = age;
+    }
+
+    public String getBreed() {
+        return breed;
+    }
+
+    public void setBreed(String breed) {
+        this.breed = breed;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
```

Comment:
&#39;if&#39; construct must use &#39;{}&#39;s.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1728147504>*